Master40.SimulationCore/Agents/CollectorAgent/Collector.Analytics.WorkSchedule.cs

```diff
@@ -98,7 +101,7 @@ private void ThroughPut(Collector agent)
             var art = from a in simulationWorkschedules
                       where a.ArticleType == "Product"
                           && a.CreatedForOrderId != null
-                          && a.Time >= lastIntervalStart
+                          && a.Time >= lastIntervalStart - 1440
```

Comment:
better use variable 
create dynamic offset

Reply:
Right, i implemented a new Parameter for this case.